starting point for first announcement (#14)
adoptium/blog.adoptium.net: content/blog/eclipse-adoptium-announcement/index.md

---
title: Eclipse Adoptium Welcomes You
date: "2021-03-23T12:00:00+00:00"
author: georgeadams
description: After months of hard work, the migration process has begun!
tags:
  - Adoptium
  - Eclipse Foundation
---

> After months of hard work, the migration process from [AdoptOpenJDK](https://adoptopenjdk.net) has begun!

Reviewer: ```suggestion
After months of hard work, the migration process from [AdoptOpenJDK](https://adoptopenjdk.net) to Eclipse Adoptium has begun!
```

> Firstly we'd like to take this oppertunity to thank the [Eclipse Foundation](https://www.eclipse.org/) for how supportive they have been throughout this entire process. Moving an OSS project with over a quarter of a billion downloads is no small feat! Especially when you throw a rebrand into the equation at the same time. There are several key changes to the project that have been summarised below.

Reviewer: ```suggestion
Firstly we'd like to take this oppertunity to thank the [Eclipse Foundation](https://www.eclipse.org/) for how supportive they have been throughout this entire process. Moving an active open source project with over a quarter of a billion downloads is no small feat! Especially when you throw a rebrand into the equation at the same time. There are several key changes to the project assets that have been summarised below.
```

### New Name & Logo
As part of the move to the Eclipse Foundation, we decided to rebrand and change the name from AdoptOpenJDK to Adoptium:

![Adoptium Logo](./adoptium_logo.png)

Many of you will have seen the new logo rollout to GitHub, Twitter, Slack and more over the last few days!

### New Locations

> Several orgs and accounts have been changed over the last few days:

Reviewer: ```suggestion
Several of our online locations have also been changed over the last few days:
```

| Account | Old Name | New Name |
| --- | --- | --- |
| Website | [adoptopenjdk.net](https://adoptopenjdk.net) | [adoptium.net](https://adoptium.net) |
| Blog | [blog.adoptopenjdk.net](https://blog.adoptopenjdk.net) | [blog.adoptium.net](https://blog.adoptium.net) |

> | GitHub | [Adoptopenjdk](https://github.com/adoptopenjdk) | [Eclipse Adoptium](https://github.com/adoptium) |

Reviewer: ```suggestion
| GitHub | [AdoptOpenJDK](https://github.com/adoptopenjdk) | [Eclipse Adoptium](https://github.com/adoptium) |
```

| Twitter | [@adoptopenjdk](https://twitter.com/adoptopenjdk) | [@adoptium](https://twitter.com/adoptium) |
| Slack | [adoptopenjdk.slack.com](https://adoptopenjdk.slack.com) | [adoptium.slack.com](https://adoptium.slack.com) |
| YouTube | [AdoptOpenJDK](https://www.youtube.com/c/AdoptOpenJDK) | [EclipseAdoptium](https://www.youtube.com/c/EclipseAdoptium) |

> 

Reviewer: ```suggestion

In most cases we have been able to leave a forward pointer from the old to the new location, but there may still be places we need to fixup.
```

### When will Adoptium begin producing binaries?

> The Adoptium working group itself will not produce any binaries. The new binaries will be produced by the [Eclipse Temurin](https://projects.eclipse.org/projects/adoptium.temurin) project which is part of the working group. It will take some time to fully migrate our build farm and code so in the short term we still encourage users to download from [AdoptOpenJDK](https://adoptopenjdk.net). The current expectation is that the first formal release will be the July PSU updates. Make sure to monitor [adoptium.net](https://adoptium.net) for more updates!

Reviewer: ```suggestion
The Adoptium working group will be promoting a number of high-quality binaries, including the group's own Temurin binaries which will be produced by the [Eclipse Temurin](https://projects.eclipse.org/projects/adoptium.temurin) project. It will take some time to fully migrate our build farm and code, so in the meantime [AdoptOpenJDK](https://adoptopenjdk.net) will continue to make binaries available as usual. The current expectation is that the first formal release of Eclipse Temurin binaries will be the July quarterly updates. Make sure to monitor this blog and [adoptium.net](https://adoptium.net) for more updates!
```

Reviewer: Adoptium won't be promoting the AdoptOpenJDK binaries, so this style of reference would be more appropriate.
